experimental/devices/cap1188/cap1188.go

```diff
@@ -0,0 +1,554 @@
+// Copyright 2017 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// Package cap1188 controls a Microchip cap1188 device over I²C.
+
+// The device is a 8 Channel Capacitive Touch Sensor with 8 LED Drivers
+//
+// Datasheet
+//
+// The official data sheet can be found here:
+//
+// http://ww1.microchip.com/downloads/en/DeviceDoc/CAP1188%20.pdf
+//
+package cap1188
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"sync"
+	"time"
+
+	"periph.io/x/periph/conn"
+	"periph.io/x/periph/conn/gpio"
+	"periph.io/x/periph/conn/i2c"
+	"periph.io/x/periph/conn/mmr"
+	"periph.io/x/periph/conn/spi"
+	"periph.io/x/periph/devices"
+)
+
+var (
+	// Debug is a debugging flag to turn on extra logging
+	Debug         bool
+	inputStatuses []TouchStatus
+)
+
+// TouchStatus is the status of an input sensor
+type TouchStatus int8
+
+func (t TouchStatus) String() string {
+	switch t {
+	case OffStatus:
+		return "Off    "
```

Comment:
Don't include whitespace here, you don't know in which context this will be called and trailing whitespaces are be unexpected from a String() method.